compiler/optimizer/OrderBlocks.cpp

```diff
@@ -164,8 +164,7 @@ bool TR_OrderBlocks::safeToMoveAway(TR::Block *block, TR::Block *prevBlock)
    if (_changeBlockOrderBasedOnHWProfile)
       return true;
 #if 1
-    if (hasOtherHotAsmFlowPredeccessors(block, prevBlock) ||
```

Comment:
It appears that this routine is related to the flags and should be folded. After this change, however, `TR_OrderBlocks::safeToMoveAway` will always return true. Should this method be folded away as well?